ci/kind/test-e2e-kind.sh

```diff
@@ -27,6 +27,7 @@ _usage="Usage: $0 [--encap-mode <mode>] [--no-proxy] [--np] [--coverage] [--help
         --no-proxy                    Disables Antrea proxy.
         --endpointslice               Enables Antrea proxy and EndpointSlice support.
         --no-np                       Disables Antrea-native policies.
+        --skip-mode-irrelevant        Skip mode irrelevant e2e cases.
```

Comment:
```suggestion
        --skip                        A comma-separated list of keywords, with which tests should be skipped.
```
to make it more generic?

Reply:
good point, just updated it.